seminar3.py

```diff
@@ -0,0 +1,56 @@
+# Дан список повторяющихся элементов. Вернуть список с дублирующимися элементами.
+# В результирующем списке не должно быть дубликатов.
+
+
+def task1():
+    my_list = [9, 17, 9, 7, 39, 2, 7, 15, 42, 15, 2]
+    res = set()
+    for el in my_list:
+        counter = my_list.count(el)

```

Comment:
Из-за этих 2-х строчек сложность вашего скрипта всегда будет квадратичной. Можно немного понизить сложность, передав в определение цикла множества.
```suggestion
    for el in set(my_list):
        counter = my_list.count(el)
```